### ISSUE TITLE
Separate post-run analysis

### ISSUE THREAD

**Issue by anufrievroman** [anufrievroman/freepaths#24]
In the future, I plan to separate the running of the simulation from the analysis so that the analysis could be performed separately with many different parameters. This will free the config files from unnecessary options, and will allow creating pretty plots without rerunning the entire simulations. Example of things that could be redone after the simulation:
- Re-plotting maps without phonons in hole, or with white patches of holes on top.
- Re-plotting distributions with desired ranges and bins.
- Re-plotting with different color maps and other cosmetics
- Changing in which time segments thermal conductivity is calculated

Most of this can be achieved just by user re-plotting things from raw data. I am not sure what would be the best way to do it? Create a separate command, or just provide a bunch of scripts for analysis, or keep all the settings in the config and allow rerunning `freepaths` without rerunning the simulation, like with `--idle` flag.

**Comment by phigas** [comment]
One challenge with this are the timeframes. The information is put into the timeframes at runtime and so it would be difficult to chantge the timeframes after the run. I propose to use a high number of timeframes during the run so that they can be summed together into different lenghts at the plotting stage. The TC calculation would then also fall into the plotting stage.

**Comment by anufrievroman** [comment]
Yes, that's precisely my current vision. Let's say we always have `NUMBER_OF_TIMEFRAMES`, which is supposed to be about 10 or so. And all profiles for each timeframe are saved. Then, we can do all:
- Show the convergence
- Output thermal conductivity calculated over last N timeframes, plus some uncertainty
- Recalculate the thermal conductivity afterward for different N if needed.
